README.md

```diff
@@ -123,6 +123,15 @@ If you want to disable the cronjob completely, run:
 
 To reenable it again simply set the `nextcloud.cron-interval` snap variable to a value that isn't `-1`
 
+#### HTTP compression configuration
+
+If desired the snap can use HTTP compression by running:
+
+    $ sudo snap set nextcloud apache.compression=true
```

Comment:
I consider Apache to be an implementation detail in the snap. In order to retain the ability to move to another web server in the future, can we think of a configuration name that doesn't leak those implementation details?

Reply:
Sure! Makes a lot of sense with hindsight...